### ISSUE TITLE
Deployment to Windows 8.1 is missing

### ISSUE THREAD

**Issue by ststeiger** [mono/SkiaSharp#270]
There's an error (Unable to load DLL 'libSkiaSharp') when publishing SkiaSharp to Windows 8.1

To reproduce:
```
dotnet restore -r win81-x64
dotnet build -r win81-x64
dotnet publish -f netcoreapp1.1 -c Release -r win81-x64
```

Is this because there is no native library for Windows 8.1 ? 
Strangely, if I copy the Windows 7 libSkiaSharp.dll into the folder, it doesn't work. 
It also doesn't work with the Windows10 libSkiaSharp.dll

Is this because there is no library for Windows 8.1 ? If so, it should be provided. Might it be because of missing C/C++-runtime files ?  If so, the C-runtime should be linked statically. 
Copying the dll for Windows 7/10 x64 or x32 to the application folder or the Windows folder didn't help either...
On Linux it began to work as soon as I compiled the native library and added it to ldconfig (BTW: there's an error when generating the documentation in the native library). 

[![Where is 8.1][1]][1]

[![Error][2]][2]

  [1]: https://i.stack.imgur.com/vhxbi.png
  [2]: https://i.stack.imgur.com/l5HR0.png

**Comment by mattleibow** [comment]
Hi @ststeiger, thanks for reporting this. There are several things here that might need addressing:

1. Deployment to `win81-x64` will be addressed in the next release. This is because `win8` does not derive from `win7`, but rather from `win` in the RID hierarchy. In the meantime, deploy to `win7` will resolve the issue. 
2. The current setup with .NET Core is to deploy to `win7` when running on Windows 7+. `win10` is not supported on versions prior to Windows 10.
3. At this point, we don't statically link the C runtime in any of the builds. See reasons and more info on #109 and #136 
4. As a result of no static C library, the MSVCRT for VS 2015 has to be installed first: https://www.microsoft.com/en-us/download/details.aspx?id=48145

So, as an action, is it possible to install the C runtime (https://www.microsoft.com/en-us/download/details.aspx?id=48145) and try running a deploy to Windows 7? If that doesn't work, then there is a bug. Deploying to Windows 8.x should be fixed in the next release.

**Comment by mattleibow** [comment]
I just did some tests and It appears that `win8` does inherit from `win7`: https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json#L50-L74

Thus, it may just be the case that the C runtime is not installed. I would first install the runtime and see if the issue goes away.

**Comment by ststeiger** [comment]
I don't have administrator rights on this machine, so I can't install it. 
But I copied the api-ms-win-crt* from c:\program files (x86)\windows kits\10\redist\ucrt\dlls\x64 from my windows 10 machine to the Win81 machine, but that didn't work. 

I tried again with the api-ms-win-crt* from AgentRansack from the Windows 10 machine to the win81 machine, **and that worked.**  So I think it's safe to assume it's the C runtime.

The files I copied (I assume it's only because of the C/C++ runtime)

19.10.2016  13:38           334 656 concrt140.dll
19.10.2016  13:38            19 648 api-ms-win-crt-conio-l1-1-0.dll
19.10.2016  13:38            22 720 api-ms-win-crt-convert-l1-1-0.dll
19.10.2016  13:38            19 136 api-ms-win-crt-environment-l1-1-0.dll
19.10.2016  13:38            20 672 api-ms-win-crt-filesystem-l1-1-0.dll
19.10.2016  13:38            19 648 api-ms-win-crt-heap-l1-1-0.dll
19.10.2016  13:38            19 136 api-ms-win-crt-locale-l1-1-0.dll
19.10.2016  13:38            27 840 api-ms-win-crt-math-l1-1-0.dll
19.10.2016  13:38            26 816 api-ms-win-crt-multibyte-l1-1-0.dll
19.10.2016  13:38            70 848 api-ms-win-crt-private-l1-1-0.dll
19.10.2016  13:38            19 648 api-ms-win-crt-process-l1-1-0.dll
19.10.2016  13:38            23 232 api-ms-win-crt-runtime-l1-1-0.dll
19.10.2016  13:38            24 768 api-ms-win-crt-stdio-l1-1-0.dll
19.10.2016  13:38            24 768 api-ms-win-crt-string-l1-1-0.dll
19.10.2016  13:38            21 184 api-ms-win-crt-time-l1-1-0.dll
19.10.2016  13:38            19 136 api-ms-win-crt-utility-l1-1-0.dll
19.10.2016  13:38            89 416 vcruntime140.dll
19.10.2016  14:21         1 232 872 ConfigLib.dll
19.10.2016  14:21           588 264 ConfigUILib.dll
19.10.2016  13:38           220 136 CrashRpt1402.dll
19.10.2016  13:38         1 558 912 dbghelp.dll
19.10.2016  14:21           590 312 DirTraverseLib.dll
19.10.2016  14:21         1 206 248 Extensions.dll
19.10.2016  13:38            47 424 MFC140CHS.dll
19.10.2016  13:38            76 096 MFC140DEU.dll
19.10.2016  13:38            66 368 MFC140ENU.dll
19.10.2016  13:38            76 096 MFC140FRA.dll
19.10.2016  13:38         5 645 112 mfc140u.dll
19.10.2016  14:21           934 376 MiscFuncLib.dll
19.10.2016  13:38           639 808 msvcp140.dll
19.10.2016  14:21           273 384 RenderLib.dll
19.10.2016  13:38         1 109 992 Scintilla.dll
19.10.2016  14:21         2 021 352 SearchLib.dll
19.10.2016  14:21         1 747 432 ShellExt.dll
19.10.2016  14:21         1 033 704 ShellExtx86.dll
19.10.2016  14:21           191 976 StreamLib.dll
19.10.2016  14:21         2 228 712 TextFuncLib.dll
19.10.2016  13:38           992 960 ucrtbase.dll
19.10.2016  14:21         1 475 048 UIExtLib.dll
